.travis.yml

```diff
@@ -80,6 +80,16 @@ matrix:
         - npm run build -- --scope "@loopback/test-repository-postgresql" --include-dependencies
         - psql -U postgres -a -f acceptance/repository-postgresql/test/schema.sql
         - cd acceptance/repository-postgresql && npm run mocha
+    - node_js: "10"
+      os: linux
+      env:
+        - TASK=test-repository-cloudant
+      install:
```

Comment:
No need for any `install` steps here at all. Just need to add `npm run docker:setup` to `scripts` any time before the last step.